doc/theory/sh_basis.rst

```diff
@@ -0,0 +1,84 @@
+.. _sh-basis:
+
+========================
+Spherical Harmonic basis
+========================
+
+Spherical Harmonics (SH) are basis functions that are used to represent and
```

Comment:
Spherical harmonics are functions defined on the sphere. A collection of SH can be used as a basis function to represent and ...

Reply:
Thanks. Addressed in 2ea22b5.